backend/tournesol/tests/test_ml_run.py

```diff
@@ -235,6 +236,7 @@ def test_tournesol_scores_different_trust(self):
             places=3,
         )
 
+    @pytest.mark.skip("Uncertainty behaves differently with BBT")
```

Comment:
Another potential simple test is:
  user1 rates +10
  user2 rates -10
  assert(score == 0)
  user1 rates +10
  assert(score > 0)